receipt_dynamo/fix_receipt_line_pylint.py

```
@@ -0,0 +1,160 @@
+        fixed_lines.append(')')
+    elif '@pytest.mark.parametrize("method_name,invalid_input,error_match"' in line:
+        # This is likely part of a multi-line parametrize
+        fixed_lines.append(line)
+    elif '"add_receipt_line", "not-a-line", "line must be an instance of ReceiptLine"' in line:
+        indent = len(line) - len(line.lstrip())
+        fixed_lines.append(line[:line.index('"line must')])
+        fixed_lines.append(' ' * (indent + 4) + '"line must be an instance of ReceiptLine"),')
+    elif '"update_receipt_line", "not-a-line", "line must be an instance of ReceiptLine"' in line:
+        indent = len(line) - len(line.lstrip())
+        fixed_lines.append(line[:line.index('"line must')])
+        fixed_lines.append(' ' * (indent + 4) + '"line must be an instance of ReceiptLine"),')
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)